pkg/models/sidecar.go

```diff
@@ -38,8 +38,10 @@ type Sidecar struct {
 	TeleporterVersion string
 	RunRelayer        bool
 	// SubnetEVM based VM's only
-	SubnetEVMMainnetChainID uint
-	ValidatorManagement     ValidatorManagementType
+	SubnetEVMMainnetChainID    uint
+	ValidatorManagement        ValidatorManagementType
+	ValidatorManagerController []string
```

Comment:
not needed, one is a constant, the other is in the genesis (really anyway both are at the genesis)